contracts/ConsensusUtils.sol

```diff
@@ -21,6 +21,7 @@ contract ConsensusUtils is EternalStorage, ValidatorSet {
   uint256 public constant SNAPSHOTS_PER_CYCLE = 0; // snapshot each 288 minutes [34560/10/60*5]
   uint256 public constant DEFAULT_VALIDATOR_FEE = 15e16; // 15%
   uint256 public constant UNBOUNDING_PERIOD = CYCLE_DURATION_BLOCKS;
+  uint256 public constant MAX_WITHDRAW_QUEUE_LENGTH = 500;
```

Comment:
we have at the moment 40K daily users. if they start using this, it could easily reach that.
i dont think this should be limited. how can this be used to attack anything?

Reply:
i guess you would need to add a way to limit the amount of records we iterate over when withdrawing, cause it might get stuck forever with out of gas

Reply:
perhaps a withdraw helper that stops if gasLeft<100K and also returns a bool indicating if reached gaslimit